src/rules/no-large-snapshots.js

```diff
@@ -10,8 +10,29 @@ const reportOnViolation = (context, node) => {
   const startLine = node.loc.start.line;
   const endLine = node.loc.end.line;
   const lineCount = endLine - startLine;
+  const whitelistedSnapshots =
+    context.options &&
+    context.options[0] &&
+    context.options[0].whitelistedSnapshots;
+
+  let isWhitelisted = false;
+  if (whitelistedSnapshots) {
+    const fileName = path.relative(process.cwd(), context.getFilename());
+    const whitelistedSnapshotsInFile = whitelistedSnapshots[fileName];
+    if (whitelistedSnapshotsInFile) {
+      const snapshotName = getStringValue(node.expression.left.property);
+      isWhitelisted =
+        whitelistedSnapshotsInFile.find(name => {
```

Comment:
use `whitelistedSnapshotsInFile.some(...` instead and avoid the `!=null` at the end

Reply:
done